apps/gitness/src/pages-v2/pull-request/pull-request-compare.tsx

```diff
@@ -175,10 +175,11 @@ export const CreatePullRequest = () => {
         repo_ref: repoRef
       },
       {
-        onSuccess: ({ body: data }) => {
+        // TODO: fix this to navigate to the new pull request after transferring a pull request page to ui
+        onSuccess: () => {
           setApiError(null)
 
-          navigate(`/spaces/${spaceId}/repos/${repoId}/pull-requests/${data?.number}`)
+          navigate(`/${spaceId}/repos/${repoId}/pulls`)
```

Comment:
quick question: did you purposely make it navigate to pr list page on pr creation because we dont have the pr details page yet?

Reply:
Yeah, I also add TODO to fix this behavior after we transfer pr details to ui package